pkg/assume/assume.go

```diff
@@ -147,7 +147,7 @@ func AssumeCommand(c *cli.Context) error {
 		configOpts.Args = assumeFlags.StringSlice("pass-through")
 	}
 
-	openBrower := !assumeFlags.Bool("env") && (assumeFlags.Bool("console") || assumeFlags.Bool("active-role") || assumeFlags.Bool("url"))
+	openBrower := !assumeFlags.Bool("env") && (assumeFlags.Bool("console") || assumeFlags.Bool("active-role") || len(assumeFlags.String("service")) > 0 || assumeFlags.Bool("url"))
```

Comment:
Nitpick: I like to use a not empty check if the any length non empty string should be true as I find it reads easier.

Buts its exactly the same check in the end

```suggestion
	openBrower := !assumeFlags.Bool("env") && (assumeFlags.Bool("console") || assumeFlags.Bool("active-role") || assumeFlags.String("service") != "" || assumeFlags.Bool("url"))
```